packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/config/portable/Template.java

```diff
@@ -50,6 +51,11 @@ public String apply( String name )
 
     public void write( File file ) throws Exception
     {
+        if ( file.getParentFile() != null )
+        {
+            new DefaultFileSystemAbstraction().mkdirs( file.getParentFile() );
```

Comment:
I'm not too happy about creating a new instance of `DefaultFileSystemAbstraction` here rather than getting a `FileSystemAbstraction` injected. Even though in this particular case we do not use a different `FileSystemAbstraction`, we might in the future...